pytezos.py

```diff
@@ -101,9 +117,8 @@ def __init__(self, key: str, passphrase: str = None, email: str = None):
                 # Dealing with secret key or seed?
                 if encoding[1] == 98:
                     self._public_key = pysodium.crypto_sign_sk_to_pk(sk=key)
-                    self._secret_key = pysodium.crypto_sign_sk_to_seed(sk=key)
```

Comment:
why remove secret key derivation when a seed is provided?

Reply:
This is the case when private key provided (length == 98) and we have already stored it on line 111.
When a seed is provided crypto_sign_seed_keypair returns a pair so we initialize public and secret keys at once.
There are basically two options: keeping the seed or the private key internally, for the sake of uniformity i've chosen the first one :)